demisto_sdk/commands/content_graph/parsers/content_item.py

```diff
@@ -242,6 +242,32 @@ def is_content_item(path: Path) -> bool:
             path
         )
 
+    def set_path(self, suffix: str) -> None:
```

Comment:
```suggestion
    def set_path_with_suffix(self, suffix: str) -> None:
```

`set_path` is a bit confusing, this just adds the suffix